code/modules/uplink/uplink_item.dm

```diff
@@ -1216,13 +1216,22 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	cost = 20
 	restricted_roles = list("Chaplain")
 	surplus = 5 //Very low chance to get it in a surplus crate even without being the chaplain
+	
 /datum/uplink_item/role_restricted/ancient_jumpsuit
 	name = "Ancient Jumpsuit"
 	desc = "A tattered old jumpsuit that will provide absolutely no benefit to you. It fills the wearer with a strange compulsion to blurt out 'glorf'."
 	item = /obj/item/clothing/under/color/grey/glorf
 	cost = 20
 	surplus = 0
 	restricted_roles = list("Assistant")
+	
+/datum/uplink_item/role_restricted/syndi_mmi
+	name = "Syndie MMI
+	desc = "A mmi modified to give cyborgs laws to serve the syndiecate without having their interface damaged by emaggs"
```

Comment:
<B>An</B> - it's "A Man machine interface" but it's "An MMI", this is because although it starts with an "M", a consonant, it starts with "em" the sound, which starts with a vowel, "e", tl;dr it's "An MMI"
<B>Syndicate</B> - not syndiecate, no extra e, and capitalised as it's an organisation's name
<B>emags</B> - single g